src/OpenSSL/crypto.py

```diff
@@ -1518,6 +1518,18 @@ def set_flags(self, flags):
         """
         _openssl_assert(_lib.X509_STORE_set_flags(self._store, flags) != 0)
 
+    def set_time(self, vfy_time):
+        """Set verification time to this store.
+
+        .. versionadded: 16.3.0
+
+        :param datetime vfy_time: The verification time to set on this store.
+        :return: ``None`` if the verification time were successfully set.
+        """
+        param = _lib.X509_VERIFY_PARAM_new()
```

Comment:
Nope, you're quite right it isn't currently exported in cryptography. Please do submit a PR there!